pages/login/index.page.tsx

```diff
@@ -0,0 +1,8 @@
+import LoginPageContainer from "@/modules/LoginPage/containers/LoginPageContainer";
```

Comment:
`LoginPageContainer` => `LoginContainer`. Please refrain from using the `Page` word from your page, container and component names